src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/CurrentOrderHeaderResponse.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.probation.courtcaseservice.controller.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.justice.probation.courtcaseservice.service.model.KeyValue;
+
+import java.time.LocalDate;
+
+@ApiModel(description = "Current order header detail")
```

Comment:
We've gone down the route of enforcing immutability on our model objects. I've done it on the PR parallel to this but it basically involves making fields final. I think you can also use the Value lombok annotation